### PR TITLE
Fix misstroke for definitely

### PR DESCRIPTION
**Pull request by jladdjr**
Confirmed that `"TKEFPBL": "definitely"` is present in `dict.json`.

Would I need to add this to https://github.com/didoesdigital/steno-dictionaries/blob/master/dictionaries/bad-habits.json as well?

### CODE REVIEW

**Comment by jladdjr** [comment]
cc @didoesdigital

**Comment by didoesdigital** [comment]
@jladdjr yes, I usually delete misstrokes from `dict.json` if they look like misstrokes as well as move them into `bad-habits.json` if it's the kind of entry people might still want around for the convenience of 'autocorrecting' misstroked words.

File: dictionaries/dict.json [hunk 1]
```diff
@@ -113325,7 +113325,6 @@
 "TKEFBGT": "defect",
 "TKEFBGT/T*EUF": "defective",
 "TKEFBGT/TOR": "defector",
-"TKEFBL": "definitely",
```
Comment (jladdjr) [inline dictionaries/dict.json]: Stripping dict.json of misstrokes

File: dictionaries/top-10000-english-words.json [hunk 2]
```diff
@@ -4368,7 +4368,7 @@
 "STRAOETS": "streets",
 "TPHAEUGS/WAOEUD": "nationwide",
 "SROERBGT": "vector",
-"TKEFPL": "definitely",
+"TKEFPBLT": "definitely",
```
Comment (jladdjr) [inline dictionaries/top-10000-english-words.json]: Per discussion on discord, going with `-LT` for consistency with "indefinitely" (`EUPB/TKEFPBLT').

**Comment by jladdjr** [comment]
@didoesdigital - updated to 1) go with `-LT` form and 2) move misstrokes from dict.json to bad-habits.json. Believe this is good-to-go!

**Pull request merged**